audio/audiowmark/Portfile

```diff
@@ -35,23 +35,21 @@ depends_build       port:pkgconfig \
                     port:autoconf-archive
 
 depends_lib         port:fftw-3-single \
+                    path:lib/libavcodec.dylib:ffmpeg \
                     port:libsndfile \
                     port:libgcrypt \
                     port:libzita-resampler \
                     port:mpg123
 
+patchfiles          patch-llvm-random.diff
+
 compiler.cxx_standard \
                     2014
-# Random generator is incompatible with LLVM 15;
-# also do not use Xcode Clang due to failing buildbots
-# https://github.com/swesterfeld/audiowmark/issues/29
-compiler.blacklist  {clang} macports-clang-15
-
-variant ffmpeg description {Build with HTTP Live Streaming support} {
-     depends_lib-append path:lib/libavcodec.dylib:ffmpeg
-     configure.args-append \
-                    --with-ffmpeg
-}
+# do not use Xcode Clang due to failing buildbots
```

Comment:
Could you be more specific? What is the error message seen on the buildbot workers?

Reply:
I can't remember as it was [added in January](https://github.com/macports/macports-ports/pull/17494). Logs has since [been deleted](https://ports.macports.org/port/audiowmark/builds/?status=failed+install-port).

Reply:
> I can't remember as it was [added in January](https://github.com/macports/macports-ports/pull/17494). Logs has since [been deleted](https://ports.macports.org/port/audiowmark/builds/?status=failed+install-port).

It looks like builds succeeded across-the-board, on a subsequent attempt:

https://ports.macports.org/port/audiowmark/details/

Reply:
> It looks like builds succeeded across-the-board, on a subsequent attempt:
> 
> https://ports.macports.org/port/audiowmark/details/

Given that all is good now, should this PR be closed?